spring-cloud-gcp-pubsub-stream-binder/src/test/java/org/springframework/cloud/gcp/stream/binder/pubsub/PubSubMessageChannelBinderTests.java

```diff
@@ -43,13 +44,15 @@
 
 	@BeforeClass
 	public static void enableTests() {
-		assumeTrue(
+		assumeTrue("PubSub Binder tests are disabled. Please enable them with -Dit.pubsub or -Dit.localdeps",
 				"true".equals(System.getProperty("it.pubsub")) || "true".equals(System.getProperty("it.localdeps")));
+		assumeThat(emulator.getEmulatorHostPort())
+				.withFailMessage("No PubSub emulator host configured").isNotNull();
```

Comment:
Now it works for me on Windows without any `gcloud`.
Nevertheless I believe we do this assumptions in the `PubSubEmulator.before()`.
It's not going to be to useful if we need to copy/paste this `enableTests()` everywhere we would like to use a `PubSubEmulator`